taxi/forms.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta:
+        model = Driver

```

Comment:
It's probably better to use `get_user_model()` here as well, even though this class is strongly dependent on **this** particular user model. But maybe it's even better to ask mentors.